homework16.py

```diff
@@ -0,0 +1,99 @@
+import requests
+from time import perf_counter
+from threading import Thread
+import multiprocessing
+from statistics import mean
+
+# Exercise 1 variant a
+result = []
+
+def request_weather(latitude, longitude):
+    response = requests.get(
+        url="https://api.open-meteo.com/v1/forecast",
+        params={
+            "latitude": latitude,
+            "longitude": longitude,
+            "hourly": "temperature_2m",
+        }
+    )
+    temperature_list = response.json()["hourly"]["temperature_2m"]
+    result.append(temperature_list)
+
+start_time = perf_counter()
+
+t1 = Thread(target=request_weather, args=(47.91, 33.39))  # Kryvyj Rih
```

Comment:
nit: I think it would be more clear to use city in dict
```
city_coordinates = [
    {"Kryvyj Rih": (47.91, 33.39)},
    {"Los Angeles": (34.05, -118.24)}
]
Thread(target=request_weather, args=city_coordinates["Kryvyj Rih"])
```

and create threads in loop for avoiding duplicate lines of code

Reply:
Thanks a lot. That's a great tip, I didn't think of that.=)